### DISCUSSION TITLE
Add mediainfo to distroless images

### DISCUSSION THREAD

**Discussion by lilws** [jesec/flood#205] [Ideas]
When access mediainfo for rtorrent, it can't spawn mediainfo and show error like this:

![image](https://user-images.githubusercontent.com/12478478/108158655-fb588100-7117-11eb-8b43-7fd620d288a4.png)

In flood:latest everything works fine. Today I tried distroless for lightweight version and find this happen. I setup distroless the same way with flood latest on docker.

**Comment by jesec** [comment]
This is a known deficiency. I am moving this issue to discussions. 

I will have to figure out a way to compile a fully static mediainfo, or use a WebAssembly version (which can be bundled into Flood).
